packages/formik/src/Formik.tsx

```diff
@@ -801,8 +803,15 @@ export function useFormik<Values extends FormikValues = FormikValues>({
     );
   });
 
-  const handleSubmit = useEventCallback(
-    (e?: React.FormEvent<HTMLFormElement>) => {
+  const handleSubmit = useEventCallback(function 
+    (e?: React.FormEvent<HTMLFormElement> | any ,extraArgs?: any) {
+
+      extraArgsOnSubmit.current=extraArgs;
```

Comment:
First of all, I was thinking the same. But this would bind the user to call the handleSubmit on the Submit Button. What if the user wants to directly call the handleSubmit then I think the above-implemented strategy would work fine.

Reply:
@probablyup a soft reminder.

Reply:
@probablyup a soft reminder.